components/textbox/index.js

```diff
@@ -29,4 +34,9 @@ const mapDispatchToProps = (dispatch) => ({
     }, dispatch),
 });
 
+const textboxLinksMapStateToProps = (state) => ({
+    isMarkdownPreviewEnabled: isFeatureEnabled(PreReleaseFeatures.MARKDOWN_PREVIEW, state),
+});
+
 export default connect(makeMapStateToProps, mapDispatchToProps, null, {withRef: true})(Textbox);
+export const TextboxLinks = connect(textboxLinksMapStateToProps)(PureTextboxLinks);
```

Comment:
This doesn't look correct since I don't think anything imports `TextboxLinks` through here. We should instead move `TextboxLinks` into its own folder and put this into an `index.js` like we do elsewhere